media/js/base/stub-attribution.js

```diff
@@ -562,14 +562,118 @@ if (typeof window.Mozilla === 'undefined') {
 
                     // Send the session ID to GA as non-interaction event.
                     if (data.client_id && data.session_id) {
+                        // UA
                         window.dataLayer.push({
                             event: 'stub-session-id',
                             eLabel: data.session_id
                         });
+                        // GA4
+                        window.dataLayer.push({
+                            event: 'stub_session_set',
+                            id: data.session_id
+                        });
+                    }
+                }
+            });
+
+            // Wait for GTM API to load so that we can pass along client and session ID
+            StubAttribution.waitForGoogleAnalytics(function () {
+                data = StubAttribution.getGAData();
+
+                if (data && StubAttribution.withinAttributionRate()) {
+                    // Send the session ID to GA as non-interaction event.
+                    if (data.client_id && data.session_id) {
+                        window.dataLayer.push({
+                            event: 'stub_session_test',
+                            label: 'session found'
```

Comment:
I don't think we need the actual values, just to know that we got them? Does that track?